.docker/package.json

```diff
@@ -0,0 +1,54 @@
+{
```

Comment:
Can we dedup this some how?  Having both places is going to make updating version and other dependencies harder

Reply:
Agreed that's where I had it last night, but this goes into my question of having a separate repo. 

Do we want to merge the adapter and docker image like it is now in this, I think it should be separate as the adapter is really a separate component

Reply:
Forgot to add, Have updated this with those changes. Life is easier now ;)